src/vidconv/vconv.c

```diff
@@ -180,6 +180,21 @@ static inline void yuv2rgb555(uint8_t *rgb, uint8_t y,
 }
 
 
+static inline void _yuv2rgb(uint8_t *rgb, uint8_t y, uint8_t u, uint8_t v)
```

Comment:
for pixelformats such as YUV420P we calculate a common color for the whole 2x2 block.
for YUV444 each pixel has its own color.

yuv2rgb I used to convert translated YUV values (int ruv, int guv, int buv)
but _yuv2rgb takes raw YUV values as input.